modules/storage/config.go

```diff
@@ -28,6 +28,7 @@ func (cfg *Config) RegisterFlagsAndApplyDefaults(prefix string, f *flag.FlagSet)
 	cfg.Trace.BlocklistPollFallback = true
 	cfg.Trace.BlocklistPollConcurrency = tempodb.DefaultBlocklistPollConcurrency
 	cfg.Trace.BlocklistPollTenantIndexBuilders = tempodb.DefaultTenantIndexBuilders
+	cfg.Trace.MaxCompactionCycle = tempodb.DefaultBlocklistPoll
```

Comment:
Should this have it's own default, or we want it tied to blocklist poll cycle?

Reply:
yeah, this is a good idea to prevent confusion.